packages/client/components/ForgotPasswordPage.tsx

```diff
@@ -131,6 +131,12 @@ const ForgotPasswordPage = (props: Props) => {
     goToPage('signin', params.toString())
   }
 
+  const errorMessage = {
+    [AuthenticationError.USER_NOT_FOUND]: 'We couldn’t find that email. Please try again.',
+    [AuthenticationError.EXCEEDED_RESET_THRESHOLD]:
+      'Too many reset password attempts. Please try again later.'
```

Comment:
add a more specific message. this isn't good because we are giving nefarious people a hint, but we'll assume most folks are good.